synapseclient/models/activity.py

```diff
@@ -55,6 +67,20 @@ class UsedURL:
     """The external URL of the file that was used such as a link to a GitHub commit
     or a link to a specific version of a software tool."""
 
+    def format_for_manifest(self) -> str:
```

Comment:
There are 2 functions with this same name and docstring, is one meant to be replaced?

Reply:
No. They're on 2 different dataclasses:
`UsedEntity` and `UsedURL`. The way the data is going to be loaded into the manifest is different so they have different implementations